app/views/leaderboard.scala.html

```diff
@@ -323,7 +329,11 @@ <h4 id="leaderboard-encouragement">
         @Html(Messages("leaderboard.encouragement.no.user"))
     } else {
         @if(UserOrgTable.getAllOrgs(user.get.userId).isEmpty) {
-            @Html(Messages("leaderboard.encouragement.no.org"))
+            @if(currentCityInfo.countryId == "taiwan") {
```

Comment:
We weren't doing this before, but after testing I'm realizing that I'd like us to open the labeling guide in a new tab (in all cases, Taiwan and otherwise)

Reply:
Sounds good, I updated all of the embedded html to open a new tab